sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/JsonSupport.java

```diff
@@ -55,6 +66,12 @@ public final class JsonSupport {
             JsonCreator.Mode.PROPERTIES));
     objectMapper.registerModule(new com.fasterxml.jackson.datatype.jdk8.Jdk8Module());
     objectMapper.registerModule(new com.fasterxml.jackson.datatype.jsr310.JavaTimeModule());
+
+    SimpleModule module = new SimpleModule();
```

Comment:
not sure if this is the right place for Done handling

Reply:
I can't image a better place for it.